packages/banner/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@hig/banner",
+  "version": "0.1.1-alpha",
```

Comment:
`0.1.0-alpha` was published for testing purposes.
`0.1.1-alpha` is currently unpublished.